cmake/Cabana.pc.cmakein

```diff
@@ -16,7 +16,7 @@
 
 # Use this in a Makefile:
 # myapp: myapp.cpp
-# 	$(CC) `pkg-config --cflags --libs Cabana` -o $@ $<
+#   $(CC) `pkg-config --cflags --libs Cabana` -o $@ $<
```

Comment:
A tab here was actually correct!

Reply:
Does this affect the correctness of this file? This looks like my editor converted tabs to spaces or something.

Reply:
This a makefile rule, which needs a tab there, having spaces is just very error-prone. We could put `<TAB>` there to make it obvious if that would make it easier.

Reply:
would this tab be to avoid copy and paste errors? I just now realized that was in a comment